course5/homework1.1/homework1.1.cc

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include <set>
+
+int main(int argc, char** argv)
+{
+	std::set<std::string> words;
```

Comment:
Using a b-tree to sort elements is a good strategy, but:
* You store std::string, each of them may require heap allocation - a very costly operation. And thy would only store copies of the strings that are already in memory; wasteful!
* By it's nature, the tree allocates every node on heap. This will be costly, especially that you know the number of elements up front. Try to write an allocator that will serve memory from pre-allcoated array